sources/packages/web/src/views/aest/SupportingUser.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <header-navigator
+    title="Back to student applications"
+    :routeLocation="{
+      name: AESTRoutesConst.STUDENT_APPLICATIONS,
+      params: { studentId },
+    }"
+    subTitle="Financial Aid Application"
+  >
+  </header-navigator>
+  <full-page-container class="my-2">
+    <formio
+      v-if="formName"
+      :formName="formName"
+      :data="formData"
+      :readOnly="true"
+    ></formio>
+    <Message severity="warn" :closable="false" v-else>
+      <strong>
+        Supporting User has not submitted the application.
+      </strong>
+    </Message>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import formio from "@/components/generic/formio.vue";
+import FullPageContainer from "@/components/layouts/FullPageContainer.vue";
+import { ref, onMounted } from "vue";
+import { SupportingUsersService } from "@/services/SupportingUserService";
+import { useFormatters } from "@/composables";
+import HeaderNavigator from "@/components/generic/HeaderNavigator.vue";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+
+export default {
+  components: {
+    formio,
+    FullPageContainer,
+    HeaderNavigator,
+  },
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+    supportingUserId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const { dateOnlyLongString } = useFormatters();
+
+    const formName = ref();
+    const formData = ref();
+
+    onMounted(async () => {
+      const supportingUsersData = await SupportingUsersService.shared.getSupportingUserData(
+        props.applicationId,
+        props.supportingUserId,
+      );
+      formName.value = supportingUsersData.formName;
+      let contactAddress = {};
+      // Here there is only one address for now
```

Comment:
supportingUsersData.contactInfo is an attribute that may or may not have an array of addresses. You may want to check for null, undefined and also that the array length is greater than 0 before the forEach loop runs.

Reply:
actually, there is a null check for `contactInfo ` (but this is no longer required)
![image](https://user-images.githubusercontent.com/77353155/155792397-7acca640-9885-4892-86bc-e24be5299a04.png) and as per the interface the `addresses` key in the `contactInfo` is not optional